python/hail/ir/renderer.py

```diff
@@ -0,0 +1,19 @@
+
+class Renderer(object):
+    def __init__(self, stop_at_jir):
+        self.stop_at_jir = stop_at_jir
+        self.count = 0
+        self.jirs = {}
+
+    def add_jir(self, jir):
+        jir_id = f'm{self.count}'
+        self.count = self.count + 1
+        self.jirs[jir_id] = jir
+        return jir_id
+
+    def __call__(self, x):
+        if self.stop_at_jir and hasattr(x, '_jir'):
+            jir_id = self.add_jir(self._jir)
+            return f'(JavaMatrix {jir_id})'
```

Comment:
I renamed Cached => Java.  Cached seems like the wrong name because it doesn't mean cached in the RDD.cache sense, or does it mean cache in the recently used sense.

Reply:
github isn't marking this unresolved after you commented - weird. 

Also, 100% agree with the name change.